runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoOverlay.java

```diff
@@ -237,5 +268,17 @@ private void renderZalcanoAOE(Graphics2D graphics, int polySize, String text, Co
 		}
 	}
 
+	private void renderPlayerCount(Graphics2D graphics)
+	{
+		Rectangle rect = new Rectangle(client.getCanvasWidth() / 2 - 15, 0, 40, 25);
```

Comment:
check out the classes in n.r.client.ui.overlay to do this a bit better.

(that's not centered btw)